.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
It would be useful if it worked.

We build for several targets with different toolchains. If you do that you will build the same thing several times since the way it works is to create a matrix with OS/compiler and set CC/CXX accordingly.  At least this was what was happening the last time.  Am I missing something?